### ISSUE TITLE
Vale a pena certificação ?

### ISSUE THREAD

**Issue by lucassantos61** [backend-br/forum#86]
Olá galera recentemente venho pesquisado sobre certificações , e estou pensado em tirar uma certificação no meu caso de PHP que é o que me agregar valor no  momento , li alguns artigos no medium que me incentivaram a tirar a certificação e estou achando valido, na opinião de vocês vocês acham valido algum tipo de certificação ?

**Comment by reginaldojunior** [comment]
Acredito que sim, ainda mais para empresas maiores que ve valor nisso. Pois é uma forma de você certificar que possui conhecimento em determinada tecnologia.

Mas isso entra no mesmo dilema da faculdade, experiência, projetos e portofolio possa te ajudar muito mais em aumentar salario ou conseguir uma vaga melhor do que um certificado. É bem relativo na verdade, depende de quem contrata e do tipo de cliente a ser adquirido.

É mais facil você provar que tem conhecimento, mas não é a unica forma de provar que o tem.

**Comment by wmartins** [comment]
Acho que a primeira pergunta que tu deve fazer é: pra que tu quer essa certificação? É para conseguir um emprego melhor? Justificar uma promoção/aumento de salário? Agregar conhecimentos?

Lembra sempre que uma certificação é um investimento (tempo e dinheiro, muitas vezes). O bom é entender qual é o retorno (seja pessoal, profissional, etc.)

Já vi empresas que valorizam pessoas certificadas, outras nem tanto.

Particularmente falando, eu não acho uma certificação um diferencial tão grande no mercado. Não sei se em concursos públicos isso faz alguma diferença, mas do que conheço do mercado de TI, ser certificado ou não acaba não provando muita coisa. Normalmente as experiências acabam contando mais.

**Comment by Fragalli** [comment]
Eu fui essa semana numa palestra do JUG Tour e o Bruno Souza respondeu uma pergunta parecida, eu concordo bastante com a visão dele.
No geral, certificação vai dar um up no seu currículo, ainda mais se tiver pouca experiência... mas não é a certificação que te transforma num profissional top de linha, é a experiência. Pode ser até um diferencial na contratação, dependendo da empresa que vc quiser entrar, mas acho que dificilmente vai ser a questão principal.
Resumindo: acho que vale a pena sim ter uma certificação, mas não dependa apenas delas para trampar num lugar legal

**Comment by lucassantos61** [comment]
@Fragalli concordo com essa visão , recentemente conversei com o Ari da comunidade de PHP e ele me falou algo bem parecido com isso que você disse certificação é válido, mas não e isso que vai dizer que você vai ser um bom profissional.

@wmartins o que eu acho mais interessante da certificação e o fato do aprendizado/reciclagem , pois o fato de uma pessoa estudar para um certificação quando ela conhece pouco da linguagem ajuda ela a aprender mais sobre e para quem já trabalha serve como uma reciclagem mesmo que ela não faça a prova, pelo menos na minha opinião.

@reginaldojunior Concordo com você , acho que a certificação não e a única maneira , e do meu ponto de vista nem a melhor maneira de provar conhecimento, mas acredito que seja um diferencial legal no currículo, e sobre a questão da faculdade e bem complicado por que eu acho que depende muito do contexto da pessoa que está inserida naquele meio, para algumas pessoas realmente e algo  que não vai agregar muito, mas para outras pessoas talvez seja uma porta de entrada para o mercado de trabalho.

**Comment by leandrotk** [comment]
Já trabalho uns 4/5 anos na área, e nunca me perguntaram sobre isso em nenhuma entrevista que fiz (e nem sobre faculdade). Todas as empresas estavam muito mais interessadas no que eu criei, no projetos e experiências que tive e o que eu fazia externamente ao trabalho (no meu caso, eu tenho feito [OSS](https://github.com/LeandroTk) e [escrevendo no meu blog](https://medium.com/@leandrotk_/)).

Agora, pra algumas empresas (principalmente as grandes - pra não dizer "apenas as grandes"), gostam (meu pai trabalha como gestor e a empresa multinacional que ele trampa, sempre vêem isso com bons olhos).

Acho que no final, certificados abrem portas (querendo ou não). Mas pergunta é: você quer entrar por essa porta? No meu caso, sempre optei por não, muito porque não tenho fit com a cultura dessas empresas grandes. Sempre achei elas engessadas, burocráticas e sem inovação, comparadas a startups (Mas essa questão é pra um outro bate-papo, acredito eu hehe).

**Comment by randercarlos** [comment]
@leandrotk 
Verdade! Em todas as vagas de emprego para programador PHP que vi: nenhuma delas exigia certificação como pré-requisito para a vaga. O que eu já vi é solicitarem certificação como diferencial.
Em todas as entrevistas de emprego que participei, o recrutadores sempre queriam saber sobre as minhas experiências, conhecimentos e, principalmente, projetos desenvolvidos e portfólios online.

Também estou pensando em obter a certificação PHP para me destacar, pois tenho o objetivo de trabalhar no exterior e obtendo essa certificação: você irá, com certeza, se diferenciar dos demais candidatos.

**Comment by Trincazul** [comment]
Também estou pensando em tirar a certificação em PHP, acredito que pra inicio de carreira é algo bom
